dev-haskell/conduit-parse/conduit-parse-0.2.1.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Parsing framework based on conduit"
+HOMEPAGE="https://github.com/k0ral/conduit-parse"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="WTFPL-2"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=dev-haskell/conduit-1.3:=[profile?]
+	dev-haskell/dlist:=[profile?]
+	dev-haskell/mtl:=[profile?]
+	dev-haskell/parsers:=[profile?]
+	dev-haskell/safe:=[profile?]
+	dev-haskell/safe-exceptions:=[profile?]
+	dev-haskell/text:=[profile?]
+	>=dev-lang/ghc-8.4.3:=
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-2.2.0.1
+	test? ( dev-haskell/resourcet
+		dev-haskell/tasty
+		dev-haskell/tasty-hunit )
+"
+
+src_configure() {
+	haskell-cabal_src_configure \
+		--flag=-enable-hlint-test
```

Comment:
Reading the [upstream cabal file](https://hackage.haskell.org/package/conduit-parse-0.2.1.0/conduit-parse.cabal), I think this should be `$(cabal_flag test enable-hlint-test)`, so that the building of the test-suite is conditional upon `FEATURES=test`. Or am I missing something here @hololeap?

Reply:
After taking a look at [this](https://github.com/k0ral/conduit-parse/blob/master/test/HLint.hs), I'd say my assumption is correct.